interface/src/app/analyze/analysis/sampleBin.js

```diff
@@ -424,12 +425,25 @@ MathFuncts, errGen) {
             MathFuncts.mean(nodeObject.activityA) -
             MathFuncts.mean(nodeObject.activityB)
           );
-          nodeObject.logsig = -Math.log10(MathFuncts.tTest(
+          nodeObject.rawPValue = MathFuncts.tTest(
             nodeObject.activityA, nodeObject.activityB
-          ).pValue());
+          ).pValue();
 
           return nodeObject;
         });
+
+        // use FDR on the raw p-values from nodeInfoSet to get adjustedPValues
+        var rawPValues = nodeInfoSet.map(function getRawPValue(nodeObject) {
+          return nodeObject.rawPValue;
+        });
+        var adjustedPValues = MathFuncts.multTest.fdr(rawPValues);
+
+        // compute logsig from the adjustedPValues & update the nodeInfoSet
+        nodeInfoSet.forEach(function(nodeObject, i) {
+          nodeObject.logsig = -Math.log10(adjustedPValues[i]);
+          nodeInfoSet[i] = nodeObject;
```

Comment:
This line seems redundant since `nodeInfoObject` is a reference to the original object in the array `nodeInfoSet`, right?

This is trivial, so please feel free to close this PR any time you feel comfortable.